docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
 
-## What kinds of applications do you want to build?
+* [Web](http://docs.asp.net/)
+  * Lean, high-performance, composable and cross-platform web, service and cloud applications with ASP.NET Core.
+* [Mobile](http://xamarin.com)
+  * Native iOS, Android, and Windows apps using existing skills, teams, and code.
+* [Console Applications](getting-started/index.md)
+  * Cloud worker and utility apps with .NET Core.
+* [Universal Windows Applications](https://developer.microsoft.com/windows)
+  * Target the richness of the Universal Windows Platform, for Windows 10.
+* [Portable Libraries](getting-started/index.md)
+   * Re-use portable libraries across cloud, mobile and desktop apps, including from NuGet.
 
-* [Console application](getting-started/index.md)
-* [Web application](http://docs.asp.net/)
+Write code in the language that suits you, C# for an imperative style, F# for a functional style or VB for productivity.
```

Comment:
We could be more up front about what language is actually supported in the current release vs. the ones we're promising for the future.